migrations/versions/e7f8a917aa8e_add_user_details_json_column.py

```diff
@@ -0,0 +1,24 @@
+"""Add user details JSON column.
+
+Revision ID: e7f8a917aa8e
+Revises: 71477dadd6ef
+Create Date: 2018-11-08 16:12:17.023569
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e7f8a917aa8e'
+down_revision = '71477dadd6ef'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('users', sa.Column('details', postgresql.JSON(astext_type=sa.Text()), server_default='{"active_at": null}', nullable=True))
```

Comment:
Maybe a better default would be `{}`? Just in case in the future we change the implementation of this feature, we don't get stuck with this default.

Reply:
Right, this is based on the default I'm setting in the models.py. I think it's okay to have those differ, I basically added it in models.py to prevent KeyError when ordering the queries.

Reply:
See my other comment: https://github.com/getredash/redash/pull/3026#discussion_r232571363 :)

Reply:
Yeah, that worked.